data/schema/account/account.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
```

Comment:
Since you have this in the `schema` package (which I think is good), you might as well put this (and all the other types you create) under the flat `data/schema` directory (ie, no need for an `account` subdirectory).